packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -546,7 +546,7 @@ describe('KeyringController', () => {
                   password,
                 );
                 expect(initialSeedWord).toBeDefined();
-                expect(initialState).toBe(controller.state);
+                expect(initialState).toStrictEqual(controller.state);
```

Comment:
I wasn't able to understand why this started failing with these changes, but this was suggested by `jest`